htdocs/web_portal/views/fragments/viewPropertiesTable.php

```diff
@@ -24,7 +24,17 @@
             ?>
                 <tr>
                     <td style="width: 35%;"><?php xecho($prop->getKeyName()); ?></td>
-                    <td style="width: 35%;"><?php xecho($prop->getKeyValue()); ?></td>
+                    <td style="width: 35%;">
+                    <?php
+                    //Adding in a hyperlink when Value is a url
+                    $value = $prop->getKeyValue();
+                    if(filter_var($value, FILTER_VALIDATE_URL)){?>
+                        <a href = <?php xecho($value); ?> ><?php xecho($value) ?></a>
+                    <?php }
+                    else{
+                        xecho($value);
+                    }?>
```

Comment:
Jumping in and out of the PHP context makes this hard to follow for me, how about something like the following?
```suggestion
<?php
    //Add a hyperlink when Value looks like a URL
    $value = $prop->getKeyValue();
    if(filter_var($value, FILTER_VALIDATE_URL)) {
        echo "<a href = ".xssafe($value).">";
        xecho($value);
        echo "</a>";
    } else {
        xecho($value);
    }
?>
```

Reply:
@sarahbyrnie, can apply @jrha's suugest and then check on a VM it still works as expected? Thanks.